admin/AmazonAI-GeneralConfiguration.php

```diff
@@ -184,9 +186,34 @@ function tim_limitless_gui(){
             $checked                = ' ';
         }
 
-        echo '<input type="checkbox" name="tim_limitless_enabled" id="tim_limitless_enabled" ' . esc_attr($checked) . ' > <p class="description"></p>';
+        echo '<input type="checkbox" name="tim_limitless_enabled" id="tim_limitless_enabled" ' . esc_attr($checked) .'onchange="toggleCheckbox(this)"' . '> <p class="description"></p>';
 
         echo '<p class="description">it is free</p>';
+        echo '<script>function toggleCheckbox(element)
+                 {
+                   if(element.checked){
```

Comment:
seems like this `if` is redundant. it's the same as `showDivs(!element.checked)`
and since it's used only for the `block` or `none` settings - might as well pass that `block`/`none` as a param

Reply:
?

Reply:
@borisjaskerovich !?

Reply:
can you write to me the code? i don't understand.

Reply:
my bad, seems like you've already changed it